prow/plugins/override/override_test.go

```diff
@@ -587,6 +600,93 @@ func TestHandle(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:    "successfully override prow job name",
+			comment: "/override prow-job",
+			contexts: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					Description: "failed",
+					State:       github.StatusFailure,
+				},
+			},
+			presubmits: map[string]config.Presubmit{
+				"prow-job": {
+					JobBase: config.JobBase{
+						Name: "prow-job",
+					},
+					Reporter: config.Reporter{
+						Context: "ci/prow/prow-job",
+					},
+				},
+			},
+			jobs: sets.NewString("ci/prow/prow-job"),
+			expected: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					State:       github.StatusSuccess,
+					Description: description(adminUser),
+				},
+			},
+		},
+		{
+			name:    "override same prow job and context",
+			comment: "/override prow-job\n/override ci/prow/prow-job",
+			contexts: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					Description: "failed",
+					State:       github.StatusFailure,
+				},
+			},
+			presubmits: map[string]config.Presubmit{
+				"prow-job": {
+					JobBase: config.JobBase{
+						Name: "prow-job",
+					},
+					Reporter: config.Reporter{
+						Context: "ci/prow/prow-job",
+					},
+				},
+			},
+			jobs: sets.NewString("ci/prow/prow-job"),
+			expected: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					State:       github.StatusSuccess,
+					Description: description(adminUser),
+				},
+			},
+		},
+		{
+			name:    "override same context and prow job",
+			comment: "/override ci/prow/prow-job\n/override prow-job",
```

Comment:
This test only differs in the order of the comment from the previous one?

Reply:
Yes. I'll make the name clearer.